source/ShipInfoDisplay.cpp

```diff
@@ -249,7 +249,7 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 	}
 	
 	// Print the number of bays for each bay-type we have
-	for(auto bayType : Ship::BAY_TYPES)
+	for(auto bayType : GameData::Category("bay type"))
```

Comment:
One of the reasons to use set as datatype for bay-types was to ensure that those categories were printed alphabetically. Do we still want to ensure printing in alphabetical order here?

Reply:
Was it particularly for the purposes of alphabetical ordered printing that a set was used, or was it just that Ship::BAY_TYPES was a set and therefore bays printed alphabetically? I think that data-defined ordering makes a bit more sense, because then bay categories can be given a "priority" in which they're printed akin to how ship categories are given priority in the shipyard. Although I do believe that the way this PR is set up right now, this will flip the way drone and fighter bays are ordered so that it lists fighters then drones instead of drones then fighters, but changing that is just a matter of flipping the definition order in the bay types category.

Reply:
My main concern is that we make an explicit choice about the order in which bays are printed (in the ships overview). I originally made sure that the bays were ordered alphabetically, but if we today decide that the order is explicitly defined by the data-files then that is also fine.
(And yes, you might want to change the order in the data-file for Fighters and Drones.)

Reply:
I've changed the bay ordering to alphabetical so that it's the same as it was before.